parser/src/rst.pest

```diff
@@ -86,10 +86,10 @@ replace = { ^"replace::" ~ " "* ~ paragraph }
 
 // Image. A directive.
 
+indent          = _{ " " | "\t" }
 image_directive = _{ ".." ~ PUSH(" "+) ~ image ~ DROP }
-image           =  { ^"image::" ~ line ~ image_opt_block? }
-image_opt_block = _{ PEEK[..-1] ~ PUSH("  " ~ POP) ~ image_option } //TODO: merge with other directives?
-image_option    =  { ":" ~ image_opt_name ~ ":" ~ line }
+image           =  { ^"image::" ~ line ~ image_option* }
+image_option    =  { indent+ ~ ":" ~ image_opt_name ~ ":" ~ line }
```

Comment:
this needs to use PEEK and so on to correctly deal with indentation.

Reply:
Can you provide a hint as to what that might have to look like?

Reply:
Thanks for the hint! Updated to use PEEK/PUSH now.